authors/apps/articles/urls.py

```diff
@@ -3,13 +3,26 @@
 from .views import (RetrieveUpdateDestroyCategory,
                     CreateListCategory,
                     CreateArticle,
-                    ArticleRetrieveUpdate)
+                    ArticleRetrieveUpdate, ChoiceView)
+from .models import LikeDislike, LikeDislikeManager
+from.models import Article
+
 
 urlpatterns = [
     path('categories/', CreateListCategory.as_view(), name='create-category'),
     path('categories/<str:slug>', RetrieveUpdateDestroyCategory.as_view(),
          name='update-delete'),
     path('articles/', CreateArticle.as_view(), name='create-articles'),
     path('article/<str:slug>/', ArticleRetrieveUpdate.as_view(),
-         name='update-articles')
+         name='update-articles'),
```

Comment:
@akram256 , Use the article slug not 'ID' in your endpoints. Most of the API spec on the readme use the same thing.

Reply:
on it

Reply:
Hey @Stanley-Okwii , I have worked on the feedback. Thank you